packages/wrangler/src/triggers/deploy.ts

```diff
@@ -258,17 +258,18 @@ export default async function triggersDeploy(
 	}
 
 	// Configure any schedules for the script.
-	// TODO: rename this to `schedules`?
-	if (triggers && triggers.length) {
+	// If schedules is not defined then we just leave whatever is previously deployed alone.
+	// If it is an empty array we will remove all schedules.
+	if (schedules) {
 		deployments.push(
 			fetchResult(`${workerUrl}/schedules`, {
 				// Note: PUT will override previous schedules on this script.
 				method: "PUT",
-				body: JSON.stringify(triggers.map((cron) => ({ cron }))),
+				body: JSON.stringify(schedules.map((cron) => ({ cron }))),
```

Comment:
Is this tested with a 0 length ?

Reply:
Thanks!
I asked because we never used length = 0 before.
(Looking at the [API](https://developers.cloudflare.com/api/resources/workers/subresources/scripts/subresources/schedules/methods/update/) now, [] must work because there is no DELETE anyway).